### PR TITLE
Fix invalid date bug on cohort view

### PR DESCRIPTION
**Pull request by clareconstantine**
Date Accepted column displays a blank cell rather than showing "Date Invalid"

### CODE REVIEW

**Comment by clareconstantine** [comment]
[Fix "Invalid Date" in the Date Accepted column for all cohort views](https://trello.com/c/hTv91kAU/560-fix-invalid-date-in-the-date-accepted-column-for-all-cohort-views)

File: apps/src/code-studio/pd/application_dashboard/cohort_view_table.jsx [hunk 1]
```diff
@@ -255,7 +255,7 @@ export class CohortViewTable extends React.Component {
   };
 
   // Format dates as abbreviated month and day, e.g. "Mar 9"
-  formatDate = (iso8601Date) => moment(iso8601Date).format("MMM D");
+  formatDate = (iso8601Date) => iso8601Date ? moment(iso8601Date).format("MMM D") : "";
```
Comment (mehalshah) [inline apps/src/code-studio/pd/application_dashboard/cohort_view_table.jsx:258]: Are you hoping to see if iso8601 exists, is defined, is truthy?

Comment (clareconstantine) [reply apps/src/code-studio/pd/application_dashboard/cohort_view_table.jsx:258]: I'm hoping to see if it exists. That parameter is the `date_accepted` value for the application, and when it's nil we were displaying "Invalid date"

Comment (mehalshah) [reply apps/src/code-studio/pd/application_dashboard/cohort_view_table.jsx:258]: Then I suggest explicitly saying `iso8601Date !== null`.  `''` evaluatees to false which may throw things off

Comment (clareconstantine) [reply apps/src/code-studio/pd/application_dashboard/cohort_view_table.jsx:258]: that wouldn't break it though, we would want to return an empty string if `iso8601Date` is `''`

Comment (clareconstantine) [reply apps/src/code-studio/pd/application_dashboard/cohort_view_table.jsx:258]: So I guess I am actually checking to see if `iso8601Date` is truthy

**Pull request merged**